### PR TITLE
responsive sign up and login page

### LINKED ISSUE #158: Login and sign up page is not responsive
## Describe the bug/feature
The login and sign up page is not responsive 

@rockingrohit9639 can i work on this issue

### PR DESCRIPTION
**Pull request by kumarishalini6**
Fixes #158

### CODE REVIEW

**Comment by kumarishalini6** [comment]
@rockingrohit9639 please have a loop on this PR

**Comment by satyampgt4** [comment]
please check login page in half screen
image
image

**Comment by kumarishalini6** [comment]
@rockingrohit9639  now check it please

File: src/components/Login/Login.css [hunk 1]
```diff
@@ -144,6 +146,50 @@
...
+  height:auto;
+ }
+
+ .loginbtn, .signupbtn{
+  padding:6px 12px;
+ }
+
+ .btns,.social_signin{
+   margin-top:10px;
+ }
```
Comment (satyampgt4) [inline src/components/Login/Login.css:190]: image

File: src/components/SignUp/SignUp.css [hunk 2]
```diff
@@ -93,4 +95,11 @@
   50% {
     transform: scale(80%);
   }
-}
+
```
Comment (satyampgt4) [inline src/components/SignUp/SignUp.css:98]: image

**Review by satyampgt4 — CHANGES_REQUESTED** [review]

**Comment by satyampgt4** [comment]
is it working in your system because i am getting error while running it

**Comment by kumarishalini6** [comment]
> is it working in your system because i am getting error while running it

let me check

File: src/components/SignUp/SignUp.css (hunk 2 above)
Comment (kumarishalini6) [reply src/components/SignUp/SignUp.css:98]: This might happen because of conflict file because this line is not available in my code

**Comment by kumarishalini6** [comment]
i have checked the whole by replacing original code this happen because of conflicts and in tab and mobile view input fields are not   not visible properly

**Comment by kumarishalini6** [comment]
just give me some time i will resolved it by tonight

**Comment by rockingrohit9639** [comment]
> just give me some time i will resolved it by tonight

okay, make sure you have pulled all the changes from main repo

**Comment by kumarishalini6** [comment]
i'll send new PR

**Pull request closed without merging**